editor/doc_tools.cpp

```diff
@@ -76,241 +76,275 @@ static String _translate_doc_string(const String &p_text) {
 	return translated.indent(indent);
 }
 
-void DocTools::merge_from(const DocTools &p_data) {
-	for (KeyValue<String, DocData::ClassDoc> &E : class_list) {
-		DocData::ClassDoc &c = E.value;
+// Comparator for constructors.
+struct ConstructorCmp {
```

Comment:
I'd rather you we spelled out `Comparator` instead of `Cmp`. Doing so also means we can get rid of the above comment.

Reply:
Agreed, got it backwards and thought this was the established wording